etc/config/csharp.amazon.properties

```diff
@@ -19,3 +19,10 @@ compiler.dotnet700csharp.clrDir=/opt/compiler-explorer/dotnet-v7.0.0
 compiler.dotnet700csharp.targetFramework=net7.0
 compiler.dotnet700csharp.buildConfig=Release
 compiler.dotnet700csharp.langVersion=latest
+
+compiler.dotnettrunkcsharp.exe=/opt/compiler-explorer/dotnet-trunk/.dotnet/dotnet
+compiler.dotnettrunkcsharp.name=.NET (main)
+compiler.dotnettrunkcsharp.clrDir=/opt/compiler-explorer/dotnet-trunk
+compiler.dotnettrunkcsharp.targetFramework=net8.0
```

Comment:
Presumably at some point we might have to update this if/when trunk moves to the next version